mindmeld/components/question_answerer.py

```diff
@@ -601,15 +633,21 @@ def _get_location(value: Union[dict, str, List]):
 
             # convert it into standard format, e.g. "37.77,122.41"
 
+            # eg. {"lat": 37.77, "lon": 122.41}
             if isinstance(value, dict) and "lat" in value and "lon" in value:
-                # eg. {"lat": 37.77, "lon": 122.41}
                 return ",".join([str(value["lat"]), str(value["lon"])])
-            elif isinstance(value, str) and "," in value and len(value.split(",")) == 2:
-                # eg. "37.77,122.41"
+            # eg. "37.77,122.41"
+            elif isinstance(value, str) and len(value.split(",")) == 2:
+                for val in value.split(","):
+                    try:
+                        val = float(val)
+                    except ValueError:
+                        # eg. "Brown, Mia"
```

Comment:
log an error here

Reply:
This is not an error due to user's KB data, it is just a check whether it is a location type string or a normal text string. I don't think a log is necessary here